verifier/verifier_test.go

```diff
@@ -1,107 +1,170 @@
-// Copyright 2021 Contributors to the Veraison project.
+// Copyright 2022 Contributors to the Veraison project.
 // SPDX-License-Identifier: Apache-2.0
 
 package verifier
 
 import (
-	"database/sql"
-	"encoding/json"
 	"fmt"
-	"io/ioutil"
 	"os"
 	"path/filepath"
-	"strings"
 	"testing"
 
 	_ "github.com/mattn/go-sqlite3"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/types/known/structpb"
+	"google.golang.org/protobuf/types/known/timestamppb"
 
 	"github.com/veraison/common"
 
 	"go.uber.org/zap"
 )
 
-func getInput(path string, v interface{}) error {
-	data, err := ioutil.ReadFile(path)
+type StubVtsClient struct {
```

Comment:
Just a note that doesn't need to be addressed now: we should probably make use of the gomock/mockgen machinery here.